feature_details.md

```diff
@@ -0,0 +1,174 @@
+
+* `duration` : The total length for this transition.
+
+* `delay` : A Document transition establishes a universal timeline for animations spanning multiple elements. This field indicates the delay in starting animations for this transition on this universal timeline.
+
+* `element` : A reference to the [Element](https://dom.spec.whatwg.org/#interface-element) in the new Document.
+
+* `sharedElement` : A reference to an element in the previous Document, explained in Referencing Elements (#referencing-elements).
+
+If one of `sharedElement` or `element` is set, this is an enter or exit transition respectively. The animations for each animatable property are defined based on `transitionType`. If both `sharedElement` and `element` are set, this is a paired transition. This transition first creates a combined image using painted content from both elements based on the `content` animation specification under [Animatable Properties](#animatable-properties). Other animatable properties are then applied to this combined output.
+
+An [open request](https://github.com/WICG/shared-element-transitions/issues/28) here is the ability to add low-level customization to these animations by specifying a delay, duration, easing curve, keyframes similar to the [Web Animation API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API). A potential option to provide this capability in future iterations is to add a `keyframes` field to this dictionary which supports property types listed under [Animatable Properties](#animatable-properties). The syntax is defined under [Processing a keyframes argument](https://drafts.csswg.org/web-animations-1/#processing-a-keyframes-argument) in Web Animations API. It’s worth mentioning that the keyframes syntax already has a concept of an implicit start/end keyframe which could map to the start/end state for each property computed in the previous/new Document respectively.
+
+#### Referencing Elements
+Specifying a transition above requires the new Document to reference elements from the previous Document. We address this problem, along with flexibility to exchange contextual information between Documents, by introducing a placeholder for shared elements captured from the previous Document. The documentTransition APIs mentioned in the following explanation are clarified under [Control Flow](#control-flow).
+
+* The dictionary used to specify a shared element cached from the previous Document in documentTransition.setSameOriginTransitions has the following fields :
+    * `element` : A reference to the [Element](https://dom.spec.whatwg.org/#interface-element) in the previous Document.
```

Comment:
Is this an actual Element that you can set properties to? I thought this would be more akin to an opaque handle, rather than an Element object, especially since it's referencing a document that is no longer active